src/preview/send.ts

```diff
@@ -0,0 +1,36 @@
+import { send as sendMail } from '../helpers/mail';
+import templates from '../templates';
+import constants from '../helpers/constants.json';
+import { rpcSuccess, rpcError } from '../helpers/utils';
+import { authenticateToken } from './utils';
+
+export default async function send(req, res) {
```

Comment:
I think for all routes we have we should use Express Router like this:
https://github.com/snapshot-labs/pineapple/blob/f1fca1a34adefab1c96cdd426d893ee88322b170/src/proxy.ts#L8-L10 this give explicit types in IDE